src/arch/x86_64/mod.rs

```diff
@@ -5,12 +5,13 @@ use align_address::Align;
 use paging::initialize_pagetables;
 use rand::Rng;
 use uhyve_interface::{GuestPhysAddr, GuestVirtAddr};
```

Comment:
Wouldn't it make sense to patch `uhyve-interface` too?

Currently, it is a bit confusing when each of  `GuestPhysAddr`, `GuestVirtAddr`, and `vm_memory::GuestAddress` should be used.

Reply:
As written below, this would result in necessary changes in the kernel, and if I can choose, I'd rather use the current address type in the kernel (and in uhyve).